CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ that users understand how the changes affect the new version.
 
 version 3.0.0-dev
 -----------------
++ Add scatterSize option to centrally control the scatter size
```

Comment:
Should the removing of normalization be mentioned in the CHANGELOG?

Reply:
Well, the adding was not... And it was not present in  the previous version.

Reply:
Ah, then it makes sense not that mention it.